.github/workflows/avm.platform.publish-module-index-json.yml

```
@@ -0,0 +1,77 @@
+# This publishes the list of all public bicep modules to an index file that the Bicep vscode extension can read for intellisense using pwsh
+name: "avm.platform.publish-module-index-json"
```

In alignment with https://github.com/Azure/bicep-registry-modules/pull/1411

```suggestion
name: .Platform - Publish [moduleIndex.json]
```